src/main/java/mate/academy/internet/shop/controller/IndexController.java

```diff
@@ -21,6 +26,7 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
+        itemService.getAll();
```

Comment:
getAll should not be here :) please, remove all test code

Reply:
sorry